sdk/src/claim.rs

```diff
@@ -381,6 +430,23 @@ impl Claim {
         self.update_manifest
     }
 
+    pub(crate) fn set_remote_manifest(&mut self, remote_url: Option<&str>) -> Result<()> {
```

Comment:
This would make more sense taking a RemoteManifest parameter without the Option, or be split into two methods, set_sidecar_manifest, and set_remote_manifest(url).
RemoteManifest could have a RemoteManifest::from_url() method.

Reply:
Split maybe, take the argument not so much.  There is another state that I don't want passed in since you can get into unintended situations.

Reply:
So I can give a remote url OR generate a sidecar, but if I don't generate a sidecar, how do I get the manifest data to post to service referenced by that Url? The claim:from_jumbf() method requires a signer which seems iffy it is a remote signer. Would it always work after committing the claim, or are there cases where we need to write to an actual file?

Reply:
Either remote option generates a sidecar.  The only difference is the embedded case has a URL embedded in the output asset.

Reply:
See the unit tests at the bottom of store.rs